### PR TITLE
Fixed links on application page

### PR DESCRIPTION
**Pull request by sofiahag**
If Mattias O. doesn't object, we can merge!

### CODE REVIEW

File: run_dir/static/css/status.css [hunk 1]
```diff
@@ -438,27 +438,6 @@ td.progress {
 	padding-top:0;
 }
 
-/* Cool colour bar for project page */
```
Comment (aanil) [inline run_dir/static/css/status.css:441]: Remove from status_b5 too!

**Comment by sofiahag** [comment]
Mattias isn't using it!

File: run_dir/design/project_samples.html [hunk 2]
```diff
@@ -18,8 +18,7 @@ <h1>
   <a href="#" class="btn btn-outline-dark btn-xs float-right" id="show_order_timeline" style="margin-top:20px;"><span id="show_orderdates_btn" style="display:none;">Show</span> <span id="hide_orderdates_btn">Hide</span> order dates on timeline</a>
```
Comment (aanil) [inline run_dir/design/project_samples.html:18]: We also don't need the Show and hide buttons on this line.

File: run_dir/design/project_samples.html [hunk 3]
```diff
@@ -18,8 +18,7 @@ <h1>
...
 </h1>
 
-<div class="project_timescale" id="project_timescale_production" style="visibility:hidden;"></div>
-<div class="project_timescale my-4" id="project_timescale_orderdates"></div>
+<div class="mt-4"><hr></div>
```
[the remark this answers is not in the document]
Comment (sofiahag) [reply run_dir/design/project_samples.html:21]: Yes, but do we want it colorful? I like it colorful, but I don't know about the others :-)

I can do a mockup and try to win everyone over!

File: run_dir/design/project_samples.html (hunk 2 above)
Comment (sofiahag) [reply run_dir/design/project_samples.html:18]: I noticed that too yesterday, already gone! (But not pushed yet) 👍

File: run_dir/design/project_samples.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (aanil) [reply run_dir/design/project_samples.html]: I like the NGI blue.

File: run_dir/design/application.html [hunk 4]
```diff
@@ -16,7 +16,7 @@ <h2 id="page_title">Projects with application <em>{{ application }}</em></h2><br
 <script>
 $.getJSON("/api/v1/application/{{ application }}", function(data) {
   $.each(data, function(k, project) {
-    $("#project_list").append('<li><a class="text-decoration-none" href="/projects/' + project + '">' + project + '</a></li>')
```
Comment (alneberg) [inline run_dir/design/application.html:19]: This part is a fix for the broken links right? 👍

**Review by alneberg — CHANGES_REQUESTED** [review]
I'm actually looking at the timescale occasionally, and I think the code is relatively well structured even if I don't understand all parts of it. Given that it's also not actually too broken, I'd say we can keep it for now.

File: run_dir/design/application.html (hunk 4 above)
Comment (sofiahag) [reply run_dir/design/application.html:19]: Yes!

**Review by aanil — APPROVED** [review]
👍

**Review by alneberg — APPROVED** [review]
Thank you! 👍

**Pull request merged**